content/blog/2016/2016-05-10-jenkins-20-vjam.adoc

```diff
@@ -347,8 +347,8 @@ link:https://wiki.jenkins.io/display/JENKINS/Ownership+Plugin[Ownership]
 plugin may be of interest though.
 
 ____
-Please consider replacing terms like "master" and "slave" with "primary" and
-"secondary".
+Please consider replacing terms like "master" and "slave" with "controller" and
+"agent".
```

Comment:
This edit seems wrong.

Reply:
Since the text was describing a meeting, I think you're right that this replacement of "primary" with "controller" and "secondary" with "agent" does not help with clarity.  Here's how it looked before:

![image](https://user-images.githubusercontent.com/156685/91863619-8b6b4280-ec2c-11ea-899f-55cfb3211340.png)

Reply:
I will put in a pr to correct this

Reply:
Still wrong.